kitsune/sumo/jinja2/handlers/403.html

```diff
@@ -1,15 +1,23 @@
 {% extends "base.html" %}
+{# L10n: The 403 Access denied page title. #}
 {% set title = _('Access denied') %}
 
 {% block content %}
   <article id="error-page" class="main sumo-page-section">
     <h1 class="sumo-page-heading">{{ title }}</h1>
     <div class="center-on-mobile">
+      {# L10n: A message displayed on the 403 Access denied page. #}
       <p>{{ _('You do not have permission to access this page.') }}</p>
 
       {% if not user.is_authenticated %}
         <p>
-          {{ _('Please try to log in / sign up first with') }} <a rel="nofollow" href="{{ url('users.fxa_authentication_init') }}">{{ _('Mozilla accounts') }}</a>.
+          {# L10n: Deprecated. A message displayed for non-authenticated users on the 403 Access denied page. #}
+          {% set deprecated_403 = _('Please try to log in / sign up first with') %}
+          {% set fxa_url = url('users.fxa_authentication_init') %}
+          {# L10n: A message displayed for non-authenticated users on the 403 Access denied page. #}
+          {% trans a_open='<a rel="nofollow" href="{{ fxa_url }}">'|safe, a_close='</a>'|safe %}
```

Comment:
I believe the `fxa_url` here will not work. Probably we need something similar to the [profile](https://github.com/mozilla/kitsune/blob/main/kitsune/users/jinja2/users/profile.html#L49)